veracruz-server/src/server.rs

```diff
@@ -16,31 +16,24 @@ use crate::platforms::icecap::VeracruzServerIceCap as VeracruzServerEnclave;
 use crate::platforms::linux::veracruz_server_linux::VeracruzServerLinux as VeracruzServerEnclave;
 #[cfg(feature = "nitro")]
 use crate::platforms::nitro::veracruz_server_nitro::VeracruzServerNitro as VeracruzServerEnclave;
-use actix_web::{dev::Server, middleware, post, web, App, HttpRequest, HttpServer};
-use base64;
-use futures::executor;
+use byteorder::{ByteOrder, LittleEndian};
 use policy_utils::policy::Policy;
-use std::{
-    sync::mpsc,
-    sync::{Arc, Mutex},
-    thread,
-};
+use std::io::{Read, Write};
+use std::net::{Shutdown, TcpListener, TcpStream};
+use std::sync::{Arc, Mutex};
+use std::thread;
 
 type EnclaveHandlerServer = Box<dyn crate::common::VeracruzServer + Sync + Send>;
 type EnclaveHandler = Arc<Mutex<Option<EnclaveHandlerServer>>>;
 
-#[post("/runtime_manager")]
-async fn runtime_manager_request(
-    enclave_handler: web::Data<EnclaveHandler>,
-    stopper: web::Data<mpsc::Sender<()>>,
-    _request: HttpRequest,
-    input_data: String,
-) -> VeracruzServerResponder {
-    let fields = input_data.split_whitespace().collect::<Vec<&str>>();
-    if fields.len() < 2 {
+fn veracruz_server_request(
+    enclave_handler: EnclaveHandler,
+    input_data: &[u8],
+) -> Result<Vec<u8>, VeracruzServerError> {
+    if input_data.len() < 4 {
         return Err(VeracruzServerError::InvalidRequestFormatError);
     }
-    let session_id = match fields[0].parse::<u32>()? {
+    let session_id = match LittleEndian::read_u32(&input_data) {
```

Comment:
Same question on the `LittleEndian` here. Just not sure if it's the right thing to do

Reply:
I've replaced with `bincode`.